env.example

```diff
@@ -21,12 +17,6 @@ EASYDITA_PASSWORD='REPLACEME!!!'
 EASYDITA_USERNAME=REPLACEME!!!
 
 SALESFORCE_API_VERSION=41.0
-SALESFORCE_ARTICLE_AUTHOR_FIELD=External_Username__c
-SALESFORCE_ARTICLE_AUTHOR_OVERRIDE_FIELD=Author_User_Id_Override__c
-SALESFORCE_ARTICLE_BODY_FIELD=Raw_HTML__c
-SALESFORCE_ARTICLE_TEXT_INDEX_FIELD=Article_Body__c
-SALESFORCE_ARTICLE_TYPE=Resource__kav
-SALESFORCE_ARTICLE_URL_PATH_PREFIX=/s/article/
```

Comment:
Please confirm it's okay for this one to change

Reply:
Good observation. The config files started out inconsistent on this URL. I've fixed it up in

59eeba5